pkg/apis/apps/v1beta1/types.go

```diff
@@ -356,6 +356,12 @@ type DeploymentStatus struct {
 	// +patchMergeKey=type
 	// +patchStrategy=merge
 	Conditions []DeploymentCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,6,rep,name=conditions"`
+
+	// Monotonically increasing counter that tracks hash collisions for
```

Comment:
Is it significant to note that this is an upper bound on collision, not the actual number of collisions?